src/store/aggregator/aggregator.sagas.ts

```diff
@@ -152,6 +211,15 @@ export function* aggregatorSaga() {
     takeLatest(aggregatorActions.setDestinationToken, fetchAllowTokenAddress),
     takeLatest(aggregatorActions.setStartingToken, fetchStartingTokenBalance),
 
+    takeLatest(aggregatorActions.setSendAmount, fetchIncentive),
+    takeLatest(aggregatorActions.setDestinationToken, fetchIncentive),
+
     takeLatest(aggregatorActions.submit, transferTokens),
   ]);
 }
+
+function roundBN(bn: BigNumber, digits: number): BigNumber {
```

Comment:
This definitely doesn't belong here. Create a new file `mathUtils.ts` (if you don't find better existing files) and move it there.